bootloader/lk.go

```diff
@@ -432,12 +440,17 @@ func (l *lk) ExtractKernelAssets(s snap.PlaceInfo, snapf snap.Container) error {
 	if err != nil {
 		return err
 	}
+	dtboPartition, err := env.FindDtboPartition(bootPartition)
+	if err != nil || dtboPartition == "" {
```

Comment:
we should have a dedicated error to know if this is not supported or check for version maybe, instead of ignoring errors?